docs/upgrading.md

```diff
@@ -154,10 +154,25 @@ Additionally there are some Custom Panels which help to add the `permission` par
 
 The following templates are overridden and should be checked for changes when upgrading Wagtail:
 
-Last checked against Wagtail version: 6.0
+Last checked against Wagtail version: 6.1
 
 - `rca/account_management/templates/wagtailadmin/base.html`
-- `rca/users/templates/wagtailusers/users/list.html`
+- ~~`rca/users/templates/wagtailusers/users/list.html`~~ This template was deleted in 2645c204425b8fa3409a110f46b2822a1953fe49 because as of Wagtail 6.1, [it's no longer used](https://github.com/wagtail/wagtail/commit/7b1644eb37b6b6cf7800276acf9abef5254fc096). Please note that the `user_listing_buttons` template tag was used in this template, and it has since been [deprecated](https://docs.wagtail.org/en/latest/releases/6.1.html#deprecation-of-user-listing-buttons-template-tag).
```

Comment:
**Suggestion:** please can this change here be highlighted to the client to test the specific functionality that should be present here. Either as a special note on the upgrade ticket or via the testing documentation we send to the client.

I tested it locally and it seems to be OK.